its/src/test/java/com/sonar/it/scanner/msbuild/TestUtils.java

```diff
@@ -261,7 +265,7 @@ private static Path getNuGetPath(Orchestrator orch) {
     return nugetPath;
   }
 
-  private static BuildResult runMSBuildQuietly(Orchestrator orch, Path projectDir, String... arguments) {
+  private static BuildResult runMSBuildQuietly(Orchestrator orch, Path projectDir, boolean simulateAzureDevopsEnvironment, String... arguments) {
```

Comment:
an alternative to passing this ugly boolean would be to pass an object `EnvironmentVariable` (to avoid confusion with `String... arguments`) - and like this it would be extendable and `var azureDevopsEnvironmentVariable = new EnvironmentVariable("SONARQUBE_SCANNER_PARAMS", "{\"sonar.dotnet.excludeTestProjects\":\"true\",\"sonar.verbose\":\"true\"}");` would be inside `testExcludedAndTest_simulateAzureDevopsEnvironmentSetting_ExcludeTestProject`